src/platform_impl/android/mod.rs

```diff
@@ -505,7 +505,7 @@ impl Window {
         let a_native_window = if let Some(native_window) = ndk_glue::native_window().as_ref() {
             unsafe { native_window.ptr().as_mut() as *mut _ as *mut _ }
         } else {
-            panic!("native window null");
+            panic!("The native window is null. Are you in between Event::Resumed and Event::Suspended?");
```

Comment:
It's totally fine to make this even more descriptive, since this message is really our first line of defense against user confusion. Currently, it doesn't really outline that this can only be called between those events, so expanding on that would be great!

Reply:
I hope it is now all informing!